src/modules/room/type/__test__/createRoomType.test.ts

```diff
@@ -0,0 +1,139 @@
+import { FastifyInstance } from "fastify";
+import { build } from "../../../../index";
+import { prisma } from "../../../../plugins/prisma";
+
+describe("POST /api/room/type", () => {
+    let fastify: FastifyInstance;
+
+    beforeAll(async () => {
+        fastify = await build();
+    });
+
+    beforeEach(async () => {
+        await prisma.hotel.deleteMany();
+        await prisma.roomType.deleteMany();
```

Comment:
Isn't this the wrong order as roomtypes belong to hotels?

Reply:
fuuuuu